ProdSuccess.md

```diff
@@ -1,40 +1,44 @@
 # What is required to deliver a product successfully?
 

```

Comment:
Initial premise was removed. 

Getting the right work done at the right time

Was that intentional?

Reply:
I added that back.